webview/.storybook/preview.ts

```diff
@@ -1,6 +1,5 @@
 import { Preview } from '@storybook/react'
 import { InternalVsCodeApi } from '../src/shared/api'
-import { action } from '@storybook/addon-actions'
```

Comment:
[F] This is the source of the bug. Having `action` wrapped around `postMessage`/`setState` in the VS Code API meant that we could record the payload sent to the extension. I have no used this in a long time and I think we should be testing that functionality differently (i.e. automated tests). I have removed it so that we can move forwards but we can park this for a while if you feel differently.

Reply:
(`@storybook/addon-actions` tries to import `@storybook/globals` which is no longer available)